examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
+
+# Reset the entrypoint
+ENTRYPOINT ["/bin/sh", "-c"]
+
+RUN yum install -y clamav-server clamav-scanner clamav-data --disablerepo=origin-local-release \
+      && mkdir -p /image-data && chmod 0777 /image-data
```

Comment:
While it's great that you provide an example clam container, it's not really useful as anything more than a starting point since it would only have the very old signatures provided in the `clamav-data` package. It may be more useful to point at a more full-featured implementation in addition to this. We're currently building a clamd pod that will be run as a daemonset on our nodes. One container in the pod runs clamd and another periodically gets updates via freshclam and other sources as well as providing a hook to have cluster-specific signatures passed in. More of an FYI than anything else. I'll send you a link to it once we're a bit further along. In the mean time, it might be worth noting the deficiencies so nobody goes off and builds something around it only to discover that they're using old signatures with no good way of updating them.